besu/src/main/java/org/hyperledger/besu/cli/options/unstable/ContractCacheOptions.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright ConsenSys AG.
```

Comment:
nit: new files should have * Copyright contributors to Hyperledger Besu  https://wiki.hyperledger.org/display/TSC/Copyright+and+License+Policy